src/components/LogoutLink.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Link from 'react-router/lib/Link';
+
+/**
+ * Logout link component
+ * @module LogoutLink
+ */
+class LogoutLink extends React.Component {
+	render() {
+		const {
+			to,
+			children,
+			...other
+		} = this.props;
+
+		return (
+			<Link
+				to={{ pathname: to, query: {logout: true} }}
```

Comment:
does the login action currently drop the logout param? thinking about the logout -> log back in flow.